libraries/botbuilder-dialogs-adaptive/src/input/textInput.ts

```diff
@@ -39,7 +39,15 @@ export class TextInput extends DialogCommand implements DialogDependencies {
     /**
      * Activity to send the user.
      */
-    public activity = new ActivityProperty();
+    public set prompt(value: string|Partial<Activity>) {
```

Comment:
This was needed because the original implementation was constructor based, but we need property based setting of this to allow declarative patterns